views/templates/front/logo_height.tpl

```diff
@@ -0,0 +1,23 @@
+{*
+ * 2011-2021 Blockonomics
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License (AFL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/afl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@blockonomics.co so we can send you a copy immediately.
+ *
+ * @author    Blockonomics Admin <admin@blockonomics.co>
+ * @copyright 2011-2021 Blockonomics
+ * @license   http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
+ * International Registered Trademark & Property of Blockonomics
+ *}
+
+<script>
+    var image = document.querySelector("img[src*='blockonomics/views/img']")
+    image.style.height = "{$blockonomicsLogoHeight|escape:'htmlall':'UTF-8'}px"
```

Comment:
We can use {$blockonomicsLogoHeight|intval} here to filter the height.

Reply:
Done